src/components/Button/Button.scss

```diff
@@ -1,34 +1,98 @@
-.btn {
-  text-decoration: none;
+:root {
+  --primary-btn-bg: #2d42fc;
+}
+
+.button {
+  cursor: pointer;
+  font-size: var(--font-sm);
   display: inline-flex;
   align-items: center;
-  background: transparent;
-  padding: 0.75rem 1.25rem;
-  color: white;
-  font-size: var(--font-sm);
-  border: 1px solid var(--dark-blue-border);
-  border-radius: var(--border-radius-sm);
-  cursor: pointer;
+  justify-content: center;
+  text-decoration: none;
+  font-weight: normal;
+  letter-spacing: 0px;
+  padding: 1.1rem 1.6rem;
+  border: none;
+  transition: all 0.2s ease-in-out;
 
-  .btn-label {
-    margin-left: 1.1625rem;
-    font-weight: normal;
-    line-height: 2.325rem;
-    letter-spacing: 0;
-    font-size: 2rem;
+  &.primary {
+    background: var(--primary-btn-bg);
+    color: #fff;
+    border-radius: 0.3rem;
+    padding: 1.65rem 1.6rem;
+    line-height: 1;
+
+    &:hover {
+      background: var(--primary-btn-hover);
+    }
+
+    &:active {
+      background-color: var(--primary-btn-active);
+    }
   }
 
-  .btn-image {
-    display: inline-flex;
-    align-items: center;
-    justify-content: center;
-    max-height: 2.3rem;
-    max-width: 2.3rem;
+  &.primary-action {
+    background: linear-gradient(90deg, #2d42fc, #2e3dcd);
+    color: #fff;
+    border-radius: 0.3rem;
+    padding: 1.65rem 1.6rem;
```

Comment:
Ideally, also standardize sizes by separate classes, e.g. ".s (small) .m (medium) .l (large) .xl etc.)

Because we can use one variant with different size i think

Reply:
Yeah I agree. I think we can keep updating the styles based on the use-case.